addons/sourcemod/scripting/AntiBhopCheat.sp

```diff
@@ -247,7 +257,7 @@ public void OnClientDisconnect(int client)
 
 public Action OnPlayerRunCmd(int client, int &buttons)
 {
-	if (!IsClientInGame(client) || IsFakeClient(client) || !IsPlayerAlive(client) || IsClientSourceTV(client))
+	if (g_bSvAutoBhop || !IsClientInGame(client) || IsFakeClient(client) || !IsPlayerAlive(client) || IsClientSourceTV(client))
```

Comment:
.BoolValue

Reply:
It's already a bool value, I guess you miss read it with `g_cvSvAutoBhop`